grpc/src/main/java/com/linecorp/armeria/server/grpc/GrpcService.java

```diff
@@ -62,6 +68,18 @@ static GrpcServiceBuilder builder() {
      */
     List<ServerServiceDefinition> services();
 
+    /**
+     * Returns a {@link Map} whose key is a route path and whose value is {@link MethodDescriptor}, which is
+     * serviced by this service.
+     */
+    default Map<String, MethodDescriptor<?, ?>> methods() {
+        return services().stream()
+                         .flatMap(service -> service.getMethods().stream())
+                         .map(ServerMethodDefinition::getMethodDescriptor)
+                         .distinct()
+                         .collect(toImmutableMap(MethodDescriptor::getFullMethodName, Function.identity()));
+    }
```

Comment:
Looks OK, but it'd be nice if we override this method in all our implementations so that it does not re-calculate the map every time.

Reply:
Overrided this method in all our implmentation.